### PR TITLE
chore: Upgrade CNI to v1.1.6

### PR DESCRIPTION
**Pull request by tamilmani1989**
**Reason for Change**:

updated cni version to latest release - 1.1.6.

**Issue Fixed**:

**Requirements**:

- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Notes**:

### CODE REVIEW

**Comment by tamilmani1989** [comment]
@sozercan can you upload the artificats to acs mirror?

windows - https://acnreleaseartifacts.azureedge.net/azure-cni/v1.1.6/binaries/azure-vnet-cni-singletenancy-windows-amd64-v1.1.6.zip

Linux - https://github.com/Azure/azure-container-networking/releases/download/v1.1.6/azure-vnet-cni-linux-amd64-v1.1.6.tgz

**Comment by sozercan** [comment]
@tamilmani1989 done

**Comment by tamilmani1989** [comment]
/assign @jackfrancis @mboersma 

can you run azp e2e?

File: vhd/packer/configure-windows-vhd.ps1 [hunk 1]
```diff
@@ -115,10 +115,9 @@ function Get-FilesToCacheOnVHD
             "https://kubernetesartifacts.azureedge.net/kubernetes/v1.19.0-beta.2/windowszip/v1.19.0-beta.2-1int.zip"
         );
         "c:\akse-cache\win-vnet-cni\" = @(
-            "https://kubernetesartifacts.azureedge.net/azure-cni/v1.1.0/binaries/azure-vnet-cni-windows-amd64-v1.1.0.zip",
```
Comment (jackfrancis) [inline vhd/packer/configure-windows-vhd.ps1:118]: I assume the above binary no longer supports any of the Windows CNI configurations?

cc @jsturtevant

Comment (tamilmani1989) [reply vhd/packer/configure-windows-vhd.ps1:118]: sorry i couldn't get it..can you explain a bit more?

Comment (jackfrancis) [reply vhd/packer/configure-windows-vhd.ps1:118]: why are we deleting the `azure-vnet-cni-windows-amd64-v1.1.0.zip` artifact from the Windows VHD configuration?

Comment (tamilmani1989) [reply vhd/packer/configure-windows-vhd.ps1:118]: the code keeps only last 3 recent versions right?

Comment (jackfrancis) [reply vhd/packer/configure-windows-vhd.ps1:118]: Just commenting that the package naming convention changed between 1.1.0 and 1.1.2. Just making sure that we have no dependencies upon the old naming convention @ 1.1.0

**Comment by tamilmani1989** [comment]
@jackfrancis / @mboersma  can you please start azp e2e? Also windows vhd pipeline failed with timeout. Please retrigger it.

File: vhd/packer/configure-windows-vhd.ps1 (hunk 1 above)
Comment (jsturtevant) [reply vhd/packer/configure-windows-vhd.ps1:118]: looks like we made the switch to the new naming convention awhile back: 

https://github.com/Azure/aks-engine/blob/efe782ed75913d4760745d49fc1eeedac45b70de/pkg/api/azenvtypes.go#L104

Likely that if someone chose 1.1.0 they would not get the correct package so removing this is probably for the better.

**Pull request merged**